src/Business Foundation/App/NoSeries/src/Setup/NoSeries.Report.al

```diff
@@ -0,0 +1,197 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.NoSeries;
+
+using System.Utilities;
+
+report 21 "No. Series"
+{
+    DefaultLayout = RDLC;
+    RDLCLayout = './NoSeries/src/Setup/NoSeries.rdlc';
```

Comment:
Move this to the new syntax in the rendering section. RDLCLayout and DefaultLauyout should be treated as being obsolete. Same comment applies to all *report.al objects